### DISCUSSION TITLE
Hygge is a word in Danish and Norwegian that describes what?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#648] [Galaxy Trivia] [answered]
*Answer with an image that best matches the description*

**Comment by DevElkami** [comment, marked as answer]
![](https://media.newyorker.com/photos/59097cbaebe912338a37891b/master/w_2560%2Cc_limit/Altman-TheYearofHyggetheDanishWayofGettingCozy-4.jpg)

**Comment by peterbaccaro** [comment]
![hygge-interiors-tips](https://user-images.githubusercontent.com/74666079/235302829-352963bc-6ad3-47aa-9fe5-3bacf22d5168.jpg)

**Comment by FleetAdmiralJakob** [comment]
![image](https://user-images.githubusercontent.com/77451351/235355868-04b91201-ef73-4dcc-9a3b-8a46edcde9c3.png)

**Comment by ParanoidUser** [comment]
Great job, guys! Viewing images for 10 seconds increases comfort level by 300%. 🤓
